lib/streamlit/web/server/browser_websocket_handler.py

```diff
@@ -134,16 +129,17 @@ def on_message(self, payload: Union[str, bytes]) -> None:
                 # "close_connection" is a special developmentMode-only
                 # message used in e2e tests to test disabling widgets.
                 if config.get_option("global.developmentMode"):
-                    self._server.stop()
+                    self._runtime.stop()
                 else:
                     LOGGER.warning(
                         "Client tried to close connection when "
                         "not in development mode"
                     )
             else:
                 # AppSession handles all other BackMsg types.
-                self._session.handle_backmsg(msg)
+                self._runtime.handle_backmsg(self._session_id, msg)
 
         except BaseException as e:
             LOGGER.error(e)
-            self._session.handle_backmsg_exception(e)
+            raise RuntimeError("TODO")
+            # self._session.handle_backmsg_exception(e)
```

Comment:
I'm not sure how we want to handle this yet, but it'll be done before merging to develop!